tools/vfsstat

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+#
+# vfsstat	Count some VFS calls.
+#		For Linux, uses BCC, eBPF. See .c file.
+#
+# Written as a basic example of counting multiple events as a stat tool.
+#
+# USAGE: vfsstat [interval [count]]
+#
+# Copyright (c) 2015 Brendan Gregg.
+# Licensed under the Apache License, Version 2.0 (the "License")
+#
+# 14-Aug-2015	Brendan Gregg	Created this.
+
+from __future__ import print_function
```

Comment:
As mentioned in another comment, the un-parenthesized print breaks in python 3. Importing print_function is a valid solution, as well as the more hackish sys.stdout = sys.stderr that I mentioned. Whichever compatibility approach you choose, please choose consistently :) Probably the import is the right one.